src/EFCore.PG/Migrations/NpgsqlMigrationsSqlGenerator.cs

```diff
@@ -676,32 +691,37 @@ public virtual void Generate(NpgsqlDropDatabaseOperation operation, [CanBeNull]
             builder.AppendLine(';');
         }
 
+        // TODO: Some forms of extension dropping are actually supported...
+        protected virtual void GenerateDropExtension(
+            [NotNull] PostgresExtension extensionType,
+            [NotNull] IModel model,
+            [NotNull] MigrationCommandListBuilder builder)
+            => throw new NotSupportedException($"Dropping extension type ${extensionType} isn't supported.");
+
+        // TODO: Some forms of extension alterations are actually supported...
+        protected virtual void GenerateAlterExtension(
+            [NotNull] PostgresExtension extensionType,
+            [NotNull] IModel model,
+            [NotNull] MigrationCommandListBuilder builder)
+            => throw new NotSupportedException($"Altering extension type ${extensionType} isn't supported.");
```

Comment:
These are not currently called, because `NpgsqlAlterDatabaseOperationAnnotations.GetPostgresExtensionsTo{Drop,Alter}` returns an empty collection.